Competitive_Programming/C++/Recursion & Backtracking/Recursion/recursion.md

```diff
@@ -0,0 +1,183 @@
+# Recursion
+
+<p> Let's have a look on the definition : </p>
+<p>A function that calls itself is known as a recursive function. And, this process is known as recursion. </p>
+
+<hr>
+
+## Working of Recursion
+
+```
+void GWOC()
+{
+    ....
+    ....
+    GWOC(); // function calling itself called recursive function
+    ....
+}
+
+// driver code
+int main()
+{
+    GWOC(); // function call
+}
+```
+
+<br>
+
+<p align = "center">
+    <img src = "https://github.com/ankushsingh24/winter-of-contributing/blob/Competitive_Programming/Competitive_Programming/C%2B%2B/Recursion%20%26%20Backtracking/Recursion/recursion.jpg" />

```

Comment:
Attach images through link. Follow below steps to generate a link for your image.

1. open any issue of any repo(create new issue).
2. in the description section, upload this image form your local.
3. After successfull upload, you'll get a link. This link will work all the time to access that image.

you can use that image link here instead of attaching through external image.

sorry, i forgot to mention in previous review...